waspc/data/Generator/templates/react-app/src/storage.ts

```diff
@@ -7,6 +8,7 @@ export type DataStore = {
 
 function createLocalStorageDataStore(prefix: string): DataStore {
     return {
+        getPrefixedKey(key) { return `${prefix}:${key}`; },
```

Comment:
Can we refactor the other functions to use this function for generating prefixed keys? I recommend defining them outside of the returned object (to avoid dealing with `this`).